updated design

tash2cool opened the pull request:
sidebar updated to include all pages of the website, private chat design modified to make easier to view

tash2cool committed:
updated design

sidebar updated to include all pages of the website, private chat design modified to make easier to view

tash2cool committed:
Merge branch 'master' into tash2

gamecoder95 reviewed (changes requested):
The adjusted design on the side bar and the private chat are awesome! There's just a few little tweaks to add:

1) IMPORTANT: Please change the link from 'surveys-students' to 'surveys' for the side-bar 'Surveys' link.

2) The font for the side-bar is a little bigger in the private chat.

3) The 'Surveys' Bar graph icon is missing in the following pages:
- survey_selection.ejs
- ss-results.ejs
- profileStudentEdit.ejs
- profileTeacherEdit.ejs
- profileStudent.ejs
- profileTeacher.ejs
- survey_teacher.ejs
- teacher_submitted.ejs

Other things (maybe can do in another commit):
- Center "Change picture" button in the profileStudent/TeacherEdit.ejs pages.
- Remove the references to the Appointment Scheduler.

tash2cool committed:
review of design

- Fixed icon of survey to appear on all pages in sidebar
- Rerouted survey and profile page links for pages with old version of sidebar
- Centered the profile upload button in edit profile page
- Added an edit user's profile link in profile page
- Fixed font size of sidebar in private-chat page
- Updated description of website in main page

tash2cool commented:
Uploaded newer version of design on branch tash2, therefore will close this pull request
